Vagrantfile

```diff
@@ -0,0 +1,69 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://atlas.hashicorp.com/search.
+  config.vm.box = "hashicorp/precise32"
+
+  # Disable automatic box update checking. If you disable this, then
+  # boxes will only be checked for updates when the user runs
+  # `vagrant box outdated`. This is not recommended.
+  # config.vm.box_check_update = false
+
+  config.vm.network "forwarded_port", guest: 3000, host: 3000
```

Comment:
Usually something in the 8XXX's (simply because it's what most stuff uses by default)

Something I usually do is use port 8080 or 8443 (depending on whether I'm using https or not), so that it's analogous to the standard 80 and 443 ports (easier to memorize)

But that makes more sense if the guest port is also 80 instead of 3000 (which will happen once you setup nginx)

So, like I said, this also works. You'll have to change this in the future anyway, once nginx and puma are setup